webhook-service/lib/common.go

```diff
@@ -54,3 +64,8 @@ func GetDeniedAlphaURLs(env map[string]string) []string {
 	}
 	return urls
 }
+
+func GetDefaultHost() string {
+	res, _ := keptnutils.ExecuteCommand("kubectl", []string{"get", "svc", "kubernetes", "-o", "jsonpath='{.spec.clusterIP}'"})
```

Comment:
pls provide an abstraction for it so that this isn't going to be executing during unit tests

Reply:
the whole thing is leftover that is not really needed